conda_docker/docker/base.py

```diff
@@ -24,6 +24,30 @@ def __init__(
         self.size = size
         self.checksum = checksum
         self.content = content
+        self.config = config or {
+            "Hostname": "",
+            "Domainname": "",
+            "User": "root",
+            "AttachStdin": False,
+            "AttachStdout": False,
+            "AttachStderr": False,
+            "Tty": False,
+            "OpenStdin": False,
+            "StdinOnce": False,
+            "Env": [
+                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+            ],
+            "Cmd": ["/bin/bash"],
+            "ArgsEscaped": True,
+            "Image": "todo-implement",
+            "Volumes": None,
+            "WorkingDir": "",
+            "Entrypoint": ["/bin/sh", "-c"],
+            "OnBuild": None,
+            "Labels": {
+                "CONDA_DOCKER": "0.0.1"
```

Comment:
This is supposed to be the conda-docker version? We maybe to should add a version bump pattern to the `rever.xsh`, if so